pom.xml

```diff
@@ -195,6 +200,14 @@
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
           <version>2.18.1</version>
+          <dependencies>
+            <dependency>
+              <!-- Force junit provider because surefire will choose the testng one if testng is detected -->
+              <groupId>org.apache.maven.surefire</groupId>
+              <artifactId>surefire-junit47</artifactId>
```

Comment:
@juherr 
Maybe as we built the TestNG and Surefire and colaborated , the .m2/repository maybe contains corrupted libs. Try to delete Maven local repo and have a look in environemnt variables. For instance CLASSPATH and JAVA_OPTS.